rocketmq-store/src/config/broker_role.rs

```diff
@@ -27,6 +27,16 @@ pub enum BrokerRole {
     Slave,
 }
 
+impl BrokerRole {
+    pub fn get_broker_role(&self) -> &'static str {
+        match self {
+            BrokerRole::AsyncMaster => "AsyncMaster",
+            BrokerRole::SyncMaster => "SyncMaster",
+            BrokerRole::Slave => "Slave",
+        }
+    }
+}
+
```

Comment:
Due to the differences in naming conventions between Rust and Java, this particular aspect should align with Java's standards.

'''
public enum BrokerRole {
    ASYNC_MASTER,
    SYNC_MASTER,
    SLAVE;
}

'''